numpy/random/mtrand/mtrand.pyx

```diff
@@ -2524,11 +2524,11 @@ cdef class RandomState:
         the probability density function:
 
         >>> import matplotlib.pyplot as plt
-        >>> import scipy.special as sps
-        >>> count, bins, ignored = plt.hist(s, 50, normed=True)
-        >>> x = np.arange(-np.pi, np.pi, 2*np.pi/50.)
-        >>> y = -np.exp(kappa*np.cos(x-mu))/(2*np.pi*sps.jn(0,kappa))
-        >>> plt.plot(x, y/max(y), linewidth=2, color='r')
+        >>> from scipy.special import i0
+        >>> plt.hist(s, 50, normed=True)
+        >>> x = np.linspace(-np.pi, np.pi, num=51)
+        >>> y = np.exp(kappa*np.cos(x-mu))/(2*np.pi*i0(kappa))
+        >>> plt.plot(x, y, linewidth=2, color='r')
```

Comment:
No, the `normed` argument of [`hist`](http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.hist) normalizes the area not the maximum.  So in http://docs.scipy.org/doc/numpy/reference/generated/numpy.random.vonmises.html the histogram has area 1.0 but the density curve has maximum 1.0, and it's mostly a coincidence that the maximum of the histogram is also near 1.0. If not for this coincidence, the problems mentioned in https://github.com/numpy/numpy/issues/5811 probably would have been found sooner.